reconcile/slack_usergroups.py

```diff
@@ -787,15 +850,13 @@ def run(
             if integration_is_enabled(QONTRACT_INTEGRATION, cluster)
         ],
     )
-    act(
+    apply_count = act(
         current_state=current_state,
         desired_state=desired_state,
         slack_map=slack_map,
         dry_run=dry_run,
     )
-    if error_occurred:
-        logging.error("Error(s) occurred.")
-        sys.exit(1)
+    return ExtendedEarlyExitRunnerResult(payload={}, applied_count=apply_count)
```

Comment:
Now we have a use case that no payload needed, we can update `ExtendedEarlyExitRunnerResult` to make it default `None` to simplify usage.

```python
    payload: object = None
```